spec/lib/feeder/feed_spec.rb

```diff
@@ -2,8 +2,47 @@
 
 describe Feeder::Feed do
   describe "#new" do
-    it "initializes" do
-      expect { Feeder::Feed.new }.not_to raise_error
+    it "sets a url" do
+      feed = Feeder::Feed.new(url: fake_feed_url)
+
+      expect(feed.url).to eq(fake_feed_url)
     end
   end
+
+  describe "#parse" do
+    it "builds entries" do
+      stub_feed
+      feed = Feeder::Feed.new(url: fake_feed_url)
+
+      feed.fetch
+
+      expect(a_request(:get, URI(fake_feed_url))).to have_been_made.at_least_once
+      expect(feed.entries.size).to eq(6)
+    end
+  end
+
+  def stub_feed
+    stub_request(:get, URI(fake_feed_url)).
+      with(
+        headers: {
+          'Accept'=>'*/*',
```

Comment:
Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.